### DISCUSSION TITLE
Form behaviour, need to reload page after save

### DISCUSSION THREAD

**Discussion by Jednadvacet** [Budibase/budibase#11325] [Help] [answered]
Hello, I would like to ask humbly for help. 
I have quite simple form with few datetime fields, some booleans, number fields and relation fields (multi and single) to log employees days of work. Relation fields values are taken from another BB internal DB tables inside application. Values are being saved to one BB internal DB table Calendar. 

When I submit the form, values are saved to DB table well - but only for the first time. After saving of data I need to reload webpage with the form to be able to save another values (row). I am trying to chain some submit button actions (Validate, Save, Clear, Navigate) but without success so far.

My setup: latest Budibase, Ubuntu self-hosted docker.

Any suggestion, please? 

Kind regards,
Ondrej

**Comment by ConorWebb96** [comment]
Hey @Jednadvacet,

Could you provide some screenshots of your form? e.g. actions attached to the button and component tree. (May require an app export to test, as I haven't been able to replicate this on my end.)

I've attached the pre-generated form block that has this functionality working. Are there any difference to your form when comparing the below screenshots.
<img width="888" alt="Screenshot 2023-07-24 at 13 11 46" src="https://github.com/Budibase/budibase/assets/126772285/cd2d3f63-40d0-4eaf-9227-11d07250824f">
<img width="887" alt="Screenshot 2023-07-24 at 13 11 51" src="https://github.com/Budibase/budibase/assets/126772285/0929f0e4-f782-40bf-8e67-712b0dd033aa">
<img width="889" alt="Screenshot 2023-07-24 at 13 11 55" src="https://github.com/Budibase/budibase/assets/126772285/05479142-5b74-4f27-be0d-f8b8afda186e">
<img width="889" alt="Screenshot 2023-07-24 at 13 12 06" src="https://github.com/Budibase/budibase/assets/126772285/13b9efc9-2c42-4b3b-ac7d-3b3275491e70">

I hope this helps.

**Reply by Jednadvacet** [reply]
Hi Connor, 
thank you for your reply. My form actions are almost the same, the only difference is "Navigate to" with exact screen "/home", but no effect if screen field is blank. Also tried to delete relation fields from form but no success.

[JobCalendar-export-1690214305720.tar.gz](https://github.com/Budibase/budibase/files/12147770/JobCalendar-export-1690214305720.tar.gz)

I send exported app to test the behaviour in your environment. 

Regards, 
Ondrej (jednadvacet)

**Reply by ConorWebb96** [reply, marked as answer]
Hey @Jednadvacet,

I believe the issue lies with how you are setting the default value. Instead of setting it to `07.00` can you set it to `2000-01-01T07:00:00.000Z`, the same applies to `15.30` can you set the default value to `2000-01-01T15:30:00.000Z`. This should fix your issue in the short term.

I do believe this is a minor bug. So I've created an issue which can be found here; https://github.com/Budibase/budibase/issues/11343

I hope this helps, thanks for raising this.

**Reply by Jednadvacet** [reply]
You're right, solved. Fantastic. Thank you for your help!!!

Regards, 
Ondrej
